### ISSUE TITLE
Wankz Network no metadata

### ISSUE THREAD

**Issue by insickin** [PAhelper/PhoenixAdult.bundle#1044]
I am tagging a scene from the Wankz Network "My Milf Boss - Exquisite MILF Veronica Avluv Gets FISTED and Fucked!" the agent finds it but doesn't fill none of the meta-data!

1. Scene URL = https://www.wankz.com/exquisite-milf-veronica-avluv-gets-fisted-and-fucked-2653448
2. File Name = My Milf Boss - Exquisite MILF Veronica Avluv Gets FISTED and Fucked!
3. Log File = [Log.log](https://github.com/PAhelper/PhoenixAdult.bundle/files/5299814/Log.log)

```
2020-09-29 16:33:15,098 (6f8fc460) :  INFO (__init__:913) - *******MEDIA TITLE****** My Milf Boss Exquisite Milf Veronica Avluv Gets Fisted and Fucked
2020-09-29 16:33:15,101 (6f8fc460) :  INFO (__init__:913) - Getting Search Settings for: My Milf Boss Exquisite Milf Veronica Avluv Gets Fisted and Fucked
2020-09-29 16:33:15,104 (6f8fc460) :  INFO (PAsearchSites:1213) - mediaTitle w/ possible abbreviation: My Milf Boss Exquisite Milf Veronica Avluv Gets Fisted and Fucked
2020-09-29 16:33:15,217 (6f8fc460) :  INFO (PAsearchSites:1457) - mediaTitle w/ possible abbrieviation fixed: My Milf Boss Exquisite Milf Veronica Avluv Gets Fisted and Fucked
2020-09-29 16:33:15,278 (6f8fc460) :  INFO (PAsearchSites:1165) - Site found with method #3
2020-09-29 16:33:15,282 (6f8fc460) :  INFO (PAsearchSites:1457) - ^^^^^^^ siteID: 600
2020-09-29 16:33:15,285 (6f8fc460) :  INFO (PAsearchSites:1457) - ^^^^^^^ Shortening Title
2020-09-29 16:33:15,290 (6f8fc460) :  INFO (PAsearchSites:1457) - my milf boss vs my milf boss
2020-09-29 16:33:15,293 (6f8fc460) :  INFO (PAsearchSites:1457) - 1
2020-09-29 16:33:15,296 (6f8fc460) :  INFO (PAsearchSites:1486) - 8
2020-09-29 16:33:15,303 (6f8fc460) :  INFO (PAsearchSites:1498) - searchTitle (before date processing): Exquisite Milf Veronica Avluv Gets Fisted and Fucked
2020-09-29 16:33:15,310 (6f8fc460) :  INFO (__init__:913) - Search Title: Exquisite Milf Veronica Avluv Gets Fisted and Fucked
2020-09-29 16:33:15,313 (6f8fc460) :  INFO (__init__:913) - Exquisite%20Milf%20Veronica%20Avluv%20Gets%20Fisted%20and%20Fucked
2020-09-29 16:33:15,385 (6f8fc460) :  INFO (PAutils:107) - Requesting GET "https://www.wankz.com/search?q=Exquisite%20Milf%20Veronica%20Avluv%20Gets%20Fisted%20and%20Fucked"
2020-09-29 16:33:15,869 (6f8fc460) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-09-29 16:33:15,873 (6f8fc460) :  DEBUG (runtime:924) - Response: [200] str, 812 bytes
2020-09-29 16:33:19,006 (6f8fc460) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNQoyMzQwNHM0CmRiaWRpMApzNwp2ZXJzaW9ubnMxMApwYXJlbnRHVUlEbnM4CnBhcmVudElEczUKTW92aWVzMTAKbWVkaWFfdHlwZXMxMjkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL2U1QmU2SnZwTnRuNDVyeW1NSmNWTjF0Y0VpeEcyam43elEyY2VuOU1MNkxkN2FVdDE4TWNLeVhGQjZSY1ZwTkd5TkpLeVhGWVRETkhZdUUza1NifDYwMD9sYW5nPWVuczQKZ3VpZHM4NwplNUJlNkp2cE50bjQ1cnltTUpjVk4xdGNFaXhHMmpuN3pRMmNlbjlNTDZMZDdhVXQxOE1jS3lYRkI2UmNWcE5HeU5KS3lYRllURE5IWXVFM2tTYnw2MDBzMgppZHIwCg__
2020-09-29 16:33:19,050 (6f8fc460) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-09-29 16:33:19,109 (6f8fc460) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNQoyMzQwNHM0CmRiaWRpMApzNwp2ZXJzaW9ubnMxMApwYXJlbnRHVUlEbnM4CnBhcmVudElEczUKTW92aWVzMTAKbWVkaWFfdHlwZXMxMjkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL2U1QmU2SnZwTnRuNDVyeW1NSmNWTjF0Y0VpeEcyam43elEyY2VuOU1MNkxkN2FVdDE4TWNLeVhGQjZSY1ZwTkd5TkpLeVhGWVRETkhZdUUza1NifDYwMD9sYW5nPWVuczQKZ3VpZHM4NwplNUJlNkp2cE50bjQ1cnltTUpjVk4xdGNFaXhHMmpuN3pRMmNlbjlNTDZMZDdhVXQxOE1jS3lYRkI2UmNWcE5HeU5KS3lYRllURE5IWXVFM2tTYnw2MDBzMgppZHIwCg__
2020-09-29 16:33:19,180 (6f8fc460) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://e5Be6JvpNtn45rymMJcVN1tcEixG2jn7zQ2cen9ML6Ld7aUt18McKyXFB6RcVpNGyNJKyXFYTDNHYuE3kSb|600?lang=en
2020-09-29 16:33:19,222 (6f8fc460) :  DEBUG (model:234) - Deserializing from /config/Library/Application Support/Plex Media Server/Metadata/Movies/2/2748fc8e5c1acac6fb64ea0cd677ffc2567197f.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2020-09-29 16:33:19,277 (6f8fc460) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/23404/tree'
2020-09-29 16:33:19,814 (6f8fc460) :  INFO (__init__:925) - ******UPDATE CALLED*******
2020-09-29 16:33:19,835 (6f8fc460) :  INFO (__init__:928) - 600
2020-09-29 16:33:19,900 (6f8fc460) :  INFO (PAutils:107) - Requesting GET "https://www.wankz.com/exquisite-milf-veronica-avluv-gets-fisted-and-fucked-2653448"
2020-09-29 16:33:20,692 (6f8fc460) :  CRITICAL (agentkit:1095) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://e5Be6JvpNtn45rymMJcVN1tcEixG2jn7zQ2cen9ML6Ld7aUt18McKyXFB6RcVpNGyNJKyXFYTDNHYuE3kSb|600?lang=en' (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-a78fef9a9/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1093, in _update
    agent.update(obj, media, lang, **kwargs)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 1540, in update
    metadata = PAsearchSites.siteQueenSnake.update(metadata, siteID, movieGenres, movieActors)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/networkWankz.py", line 32, in update
    metadata.title = detailsPageElements.xpath('//h1[@class="detail__title"]')[0].text_content().strip()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-a78fef9a9/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2020-09-29 16:33:20,729 (6f8fc460) :  DEBUG (model:229) - Serializing to /config/Library/Application Support/Plex Media Server/Metadata/Movies/2/2748fc8e5c1acac6fb64ea0cd677ffc2567197f.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2020-09-29 16:33:20,790 (6f8fc460) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-09-29 16:33:20,795 (6f8fc460) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

**Comment by DirtyRacer1337** [comment]
Try with 138b334b027ce080efa2da32d28ddafa7c77d973

**Comment by insickin** [comment]
Thanks, it now fills almost everything only the cast is not filled, can this be fixed?

**Comment by DirtyRacer1337** [comment]
Stuped error :facepalm:, fixed in b8429f7d67650742c1277ecc3dc6ee3401f42b24

**Comment by insickin** [comment]
Tried your fix/latest update but it still doesn't work :(

**Comment by DirtyRacer1337** [comment]
Another stupid error :facepalm::facepalm:, fixed in 9981665d8f8922d768ae596e33539e6274d967d8, now it's 100%

**Comment by insickin** [comment]
Thanks, that fixed it!
